spec/tom_queue/queue_manager_spec.rb

```diff
@@ -81,7 +81,7 @@
       end
 
       it "should throw an ArgumentError exception if :run_at isn't a Time object" do
-        lambda { 
+        lambda {
           manager.publish("future", :run_at => "around 10pm ?")
         }.should raise_exception(ArgumentError, /must be a Time object/)
```

Comment:
Dude, it's a whitespace change... I could go through it all and update the syntax if you like, but until it's deprecated then it seems out of scope for a version upgrade.